src/dropck.md

```diff
@@ -70,30 +75,38 @@ impl<'a> Drop for Inspector<'a> {
     }
 }
 
+struct World<'a> {
+    inspector: Option<Inspector<'a>>,
+    days: Box<u8>,
+}
+
 fn main() {
-    let (inspector, days);
-    days = Box::new(1);
-    inspector = Inspector(&days);
+    let mut world = World {
+        inspector: None,
+        days: Box::new(1),
+    };
+    world.inspector = Some(Inspector(&world.days));
     // Let's say `days` happens to get dropped first.
     // Then when Inspector is dropped, it will try to read free'd memory!
 }
 ```
 
 ```text
-error[E0597]: `days` does not live long enough
-  --> src/main.rs:12:28
+error[E0597]: `world.days` does not live long enough
+  --> /tmp/.private/vorner/mdbook-Br8SZ0/dropck.md:88:39
```

Comment:
could you revert this line (for cleanliness)

Reply:
Yes, sure (I assume you mean only the one with the file, I'm leaving the addition of `world` in)